src/index.ts

```diff
@@ -396,6 +396,16 @@ export function normalize(input: number | string): string {
   return ethUtil.addHexPrefix(input.toLowerCase());
 }
 
+/**
+ * Create an Ethereum-specific signature for a message.
+ *
+ * This function is equivalent to the `eth_sign` Ethereum JSON-RPC method as specified in EIP-1417,
+ * as well as the MetaMask provider `personal_sign` method.
```

Comment:
I am on a crusade to make "provider" a proscribed term, at least until we can agree on what it means!

```suggestion
 * as well as the MetaMask's `personal_sign` method.
```

Reply:
Sure! This still reads fine without that term.